### PR TITLE
VAGOV-1214: va-socks-proxy trigger builds from Drupal

### PR DESCRIPTION
**Pull request by ElijahLynn**
This PR has a change to .lando.yml that will require the ENTIRE engineering team to upgrade to Lando 3.0.0-rc9 or higher, we must coordinate before merging.
As long as the correct ENV variables are exported, a `lando start|rebuild` will now bring up a fully working SOCKS proxy and will support triggering builds locally. See /README.md for more information.

This PR on the devops side, https://github.com/department-of-veterans-affairs/va.gov-cms-devops/pull/23, needs to be merged first, then we can remove the 'devops' and 'skip-ci' labels and it should build successfully. 

peek 2019-02-12 20-32

### CODE REVIEW

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

File: docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.module [hunk 1]
```diff
@@ -12,8 +12,6 @@ use Drupal\Core\Form\FormStateInterface;
  * Implements hook_form_alter().
  */
 function va_gov_build_trigger_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  // This is our preview button on node forms.
-  $host = \Drupal::request()->getHost();
```
Comment (ethanteague) [inline docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.module]: Nice catch - thanks for cleaning up my mess, @ElijahLynn  :+1:

Comment (ElijahLynn) [reply docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.module]: We can thank IntelliJ for that catch ;).

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by mbenziane** [comment]
Created an environment devops-vagovcms#pr-137<br><br> [button_visit-site](http://prdevops-137.va.agile6.com)

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
PR for devops side has been made https://github.com/department-of-veterans-affairs/va.gov-cms-devops/pull/23

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
rebuild

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
Passing corresponding devops PR is here http://jenkins-ecs-master-1574396032.us-gov-west-1.elb.amazonaws.com/job/ecs_ci_devops/78/

**Comment by ElijahLynn** [comment]
We need to merge this CMS PR, https://github.com/department-of-veterans-affairs/va.gov-cms/pull/137, during working hours. Once we merge devops PR we remove `skip-ci` & `devops` labels from it, build again for a pass using devops' `master` + the CMS PR branch. Then as soon as it is merged all developers need to update to lando 3.0.0rc9+ when they pull the latest `develop` branch down. Then they need to rebase all their branches off of `develop` e.g. `git rebase develop` because otherwise their `lando *` commands will stop working because .lando.yml changed.

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
Odd, still thinks it has label 
image

**Comment by ElijahLynn** [comment]
Payload says it has zero labels: 
image

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
finally worked

**Comment by mbenziane** [comment]
Created an environment test-vagovcms#pr-137<br><br> [button_visit-site](http://pr-137.va.agile6.com)

**Comment by ElijahLynn** [comment]
Need to merge this as it is causing DEV deploys to fail now that the devops code has been merged in.

**Pull request merged**